### PR TITLE
Rename 'service' to 'action' in automations and scripts

### PR DESCRIPTION
**Pull request by frenck**
## Proposed change

This renames `service` to `action` to perform service actions. 

Before:
 
```yaml
- service: light.turn_on
  target:
    area: living_room
```

After:

```yaml
- action: light.turn_on
  target:
    area: living_room
```

⚠️ This is fully backward compatible; no existing automations or scripts will break. The `service` key will still work and **IS NOT** deprecated by this. We keep it around just like we still support `service_template`.

## Type of change

- [ ] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logos and icons in [Brands repository](https://github.com/home-assistant/brands).
- [x] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: https://github.com/home-assistant/core/pull/122845
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: fixes #

## Checklist

- [x] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [x] The documentation follows the Home Assistant documentation [standards].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

**Review by c0ffeeca7 — DISMISSED** [review]
Nice :star_struck: Thank you, @frenck :+1:

File: source/_integrations/aws.markdown [hunk 1]
```diff
@@ -38,7 +38,7 @@ aws:
       aws_secret_access_key: AWS_SECRET
   notify:
     # use the first credential defined in aws integration by default
-    - service: lambda
+    - action: lambda
```
Comment (ludeeus) [inline source/_integrations/aws.markdown]:
```suggestion
    - service: lambda
```
This has not changed in the `aws` integration schema.

**Review by ludeeus — CHANGES_REQUESTED** [review]

File: source/_integrations/aws.markdown (hunk 1 above)
Comment (frenck) [reply source/_integrations/aws.markdown]: Nice catch!

**Review by ludeeus — APPROVED** [review]
LGTM 👍

**Pull request merged**